Tests/CXFoundationTests/KeyValueObservingSpec.swift

```diff
@@ -0,0 +1,135 @@
+import CXShim
+import CXTestUtility
+import Foundation
+import Nimble
+import Quick
+
+#if canImport(FoundationNetworking)
+import FoundationNetworking
+#endif
+
+class KeyValueObservingSpec: QuickSpec {
+    #if swift(>=5.1) && canImport(ObjectiveC)
```

Comment:
Bypass this test for system Combine? They does't adapt KVO publisher yet.
```suggestion
    #if swift(>=5.1) && canImport(ObjectiveC) && !USE_COMBINE
```

Reply:
Foundation includes the KVO publisher, I believe since Xcode 11.0. Certainly they are in Xcode 11.3.1 which is the oldest version I have handy.

Reply:
These tests pass in the "combine" CI configuration.

Reply:
That's odd... I can use it (in both Xcode 11.3.1 and 11.4.1), but [official documentation](https://developer.apple.com/documentation/objectivec/nsobject/keyvalueobservingpublisher) says `KeyValueObservingPublisher` does't conforms to `Publisher`. ⌘+Click in Xcode does't bring me to the declaration either.

Reply:
OK CI failed elsewhere. It seems Combine changed its behavior once again.

Reply:
Xcode and the documentation don't say much about the KVO publisher because it involves `_KeyValueCodingAndObservingPublishing`, which starts with an underscore. Apple doesn't document, and Xcode hides, SDK identifiers that start with `_`.

Reply:
Yes `_KeyValueCodingAndObservingPublishing.publisher(for:options:)` is underscored and hidden. `KeyValueObservingPublisher.didChange` and conformance of `Publisher` don't involve underscore though.

I've seen similar header magic before (#29) and I still don't understand why.